odoo/modules/loading.py

```diff
@@ -419,6 +419,9 @@ def load_modules(registry, force_demo=False, status=None, update_module=False):
         if not graph:
             _logger.critical('module base cannot be loaded! (hint: verify addons-path)')
             raise ImportError('Module `base` cannot be loaded! (hint: verify addons-path)')
+        if update_module:
```

Comment:
NIT make it semantically more correct if you define `(ignored unless -u)` in the CLI
`if update_module and tools.config['update']`